File: lab/c/somas/soma_simples.c
#include <stdio.h>

// Versão 1
int soma1(int x, int y){
    return x + y;
}

int soma2(int x, int y){
    int soma = x + y;
    return soma;
}


int main(){
    int x, y, soma;

    printf("X: ");
    scanf("%d", &x);

    printf("Y: ");
    scanf("%d", &y);

    // Versão 3
    soma = x + y;
    printf("Soma: %d\n", soma);

    // Chamando outras versões
    soma = soma1(x, y);
    printf("Soma: %d\n", soma);

    printf("Soma: %d\n", soma2(x, y));
}

File: lab/javascript/somas/soma_simples.js
// Versão 1
function soma1(x, y){
    // console.log(x + y);
    return x + y;
}


// Versão 2
function soma2(x, y){
    const soma = x + y;
    // console.log(soma);
    return soma;
}


// Versão 3
const prompt = require('prompt-sync')();
let x = parseFloat(prompt("X: "));
let y = parseFloat(prompt("Y: "));

let soma = x + y;
console.log("Soma:", soma);

// Chamando outras verções
soma = soma1(x, y);
console.log(`Soma: ${soma}`);

console.log(`Soma: ${soma2(x, y)}`);
